server/datastore/mysql/vpp.go

```diff
@@ -117,9 +117,9 @@ func (ds *Datastore) GetSummaryHostVPPAppInstalls(ctx context.Context, teamID *u
 	// Currently there is no host_deleted_at in host_vpp_software_installs, so
 	// not handling it as part of the unified queue work.
 
-	// TODO(uniq): refactor vppHostStatusNamedQuery to use the same logic as below
+	// TODO(sarah): refactor vppHostStatusNamedQuery to use the same logic as below
```

Comment:
@gillespi314 I'm not sure what you mean here, the `vppAppHostStatusNamedQuery` helper computes the status the same way AFAICT and it is still used in the ListHostSoftware mega-query.

Reply:
I was mainly concerned with ensuring consistency with whatever approach we ended up with regarding the group-wise max and whether we're using timestamp comparisons or staying with a purely id-based approach so feel free to remove this if you think we're good there.